fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -4014,6 +4014,13 @@ public void alterTableProperties(Database db, OlapTable table, Map<String, Strin
         if (logProperties.containsKey(PropertyAnalyzer.PROPERTIES_PARTITION_LIVE_NUMBER)) {
             tableProperty.getProperties().put(PropertyAnalyzer.PROPERTIES_PARTITION_LIVE_NUMBER,
                     String.valueOf(partitionLiveNumber));
+            if (partitionLiveNumber == TableProperty.INVALID) {
+                GlobalStateMgr.getCurrentState().getDynamicPartitionScheduler().removeTtlPartitionTable(db.getId(),
+                        table.getId());
+            } else {
+                GlobalStateMgr.getCurrentState().getDynamicPartitionScheduler().registerTtlPartitionTable(db.getId(),
+                        table.getId());
+            }
             tableProperty.setPartitionTTLNumber(partitionLiveNumber);
         } else if (logProperties.containsKey(PropertyAnalyzer.PROPERTIES_STORAGE_MEDIUM)) {
```

Comment:
I'm confused. `else if` means only one of these if/else clauses will work.
Could you tell me the reason?

Reply:
If you can only handle one property, there is no error to notify users. Users probably think all changes take effect.

Reply:
You are right, this does not take into account the situation where the user uses multiple attributes at the same time and takes effect at the same time. I will change the code here.